internal/namespaces/instance/v1/custom_server_create_test.go

```diff
@@ -385,6 +426,26 @@ func Test_CreateServerErrors(t *testing.T) {
 		DisableParallel: true,
 	}))
 
+	t.Run("Error: invalid root volume snapshot ID", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance server create image=ubuntu_bionic root-volume=local:29da9ad9-e759-4a56-82c8-f0607f93055c",
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(1),
+		),
+		DisableParallel: true,
+	}))
+
+	t.Run("Error: invalid additional volume snapshot ID", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance server create image=ubuntu_bionic additional-volumes.0=block:29da9ad9-e759-4a56-82c8-f0607f93055c",
```

Comment:
Why is this id hard coded?

Reply:
This is a test that should output an error